react/filter-items-from-list/src/filter.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Filter(props) {
```

Comment:
You can do the destructure directly in the function definition:
```js
export default function Filter({filter, setFilter}) {
```

Reply:
done